tests/dragonfly/cluster_test.py

```diff
@@ -1433,7 +1433,7 @@ async def test_migration_with_key_ttl(df_factory):
     assert await nodes[1].client.execute_command("stick k_sticky") == 0
 
 
-@dfly_args({"proactor_threads": 4, "cluster_mode": "yes", "serialization_max_chunk_size": 0})
```

Comment:
I dont really understand how this is passing now.. you did this change setting serialization_max_chunk_size=0 because they failed when you created this big value serialization PR with disabling the compression

Reply:
For some reason I thought they got fixed but they failed:
https://github.com/dragonflydb/dragonfly/actions/runs/12182325655/job/33981070627#step:12:1737 here and also cascaded (I introduced this https://github.com/dragonflydb/dragonfly/commit/f9f93b108c5b44ddd3c8b588b24226345640d347 to mitigate this effect).

I will revert this